openscad.pro

```diff
@@ -194,10 +194,7 @@ CONFIG += hidapi
 CONFIG += spnav
 CONFIG += double-conversion
 
-#Uncomment the following line to enable the QScintilla editor
-!nogui {
```

Comment:
I am not sure if this is the perfect solution, however, I don't see much harm in it. This change instructs to always enable scintilla. I think that failing the travis check without this change is more of a travis configuration issue since the application builds nicely on my local ubuntu system.